matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/util/StringProvider.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2019 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.matrix.android.internal.util
+
+import android.content.res.Resources
+import androidx.annotation.NonNull
+import androidx.annotation.StringRes
+
+class StringProvider(private val resources: Resources) {
```

Comment:
This class already exist in the vector module, please make it internal to avoid confusion with import

Reply:
Ok, thought that putting it in internal package was enough. Modifier added

Reply:
`internal` package name is just a convention :-)